src/docs/release-notes/R33/FullNotes.mdx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+## Reusable Workflows
+
+-   Developed Okta authentication workflow for both React and React Native.
+-   Developed a plan/architecture for a set of workflows around user management that could be dropped-in to product team applications.
+
+## Brightlayer Charts
+
+-   Migrated Brightlayer Charts Storybook to Brightlayer development documentation.
```

Comment:
This isn't quite true...we added their code to our blui-community packages and created a new documentation project, but it's not hosted anywhere yet.
```suggestion
-   Innersourced Brightlayer Charts code into the BLUI community packages (LINK).
```